tests/hotplug/memory.go

```diff
@@ -290,6 +290,24 @@ var _ = Describe("[sig-compute][Serial]Memory Hotplug", decorators.SigCompute, d
 			_, err = virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patchBytes, k8smetav1.PatchOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
+			By("Waiting for HotMemoryChange condition to appear")
+			Eventually(ThisVMI(vmi), 1*time.Minute, 2*time.Second).Should(HaveConditionTrue(v1.VirtualMachineInstanceMemoryChange))
+
+			By("Ensuring live-migration started")
+			var migration *v1.VirtualMachineInstanceMigration
+			Eventually(func() bool {
+				migrations, err := virtClient.VirtualMachineInstanceMigration(vm.Namespace).List(context.Background(), k8smetav1.ListOptions{})
```

Comment:
You can filter migrations by `"kubevirt.io/vmi-name"` label with `vmi.Name` as a value, saving all the logic below.

Reply:
Done.